src/main/java/handler/DataHandler.java

```diff
@@ -0,0 +1,85 @@
+package handler;
+import dukeexecpetions.DeadlineEmptyException;
+import dukeexecpetions.InvalidCmd;
+import items.Items;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+/** This class represents the loading, adding and deleting of tasks Sir Duke manages*/
+public class DataHandler {
+    public enum TaskTag {
+        T, // Todo
+        D, // Deadline
+        E // Event
+    }
+    public static void loadData(Items items)
+            throws IOException {
+        File data = new File("data/sirDuke.txt");
+        Scanner parser;
+        try {
+            parser = new Scanner(data);
+        } catch (FileNotFoundException e) {
+            File dir = new File("data");
+            if (dir.exists()) {
+                data.createNewFile();
+            } else {
+                dir.mkdir();
+                data.createNewFile();
+            }
+            parser = new Scanner(data);
+        }
+        while (parser.hasNext()) {
+            // parse through the text
+            String line = parser.nextLine();
+            String[] inputsToCmd;
+            TaskTag tag;
+            Boolean isDone;
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMM d yyyy HH:mm");
+            try {
+                tag = TaskTag.valueOf(String.valueOf(line.charAt(0)));
```

Comment:
Could it enhance readability to split this code in the block into logical units separated by one blank line?